Massive Map overhaul

Bob-Murphy opened the pull request:
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [ ] Enhancement
4. [X] All of the above
### What have you changed and why?
Information:
Things!
Check the screenshot as well as the tickets this is closing for more context.
![image](https://user-images.githubusercontent.com/50651578/190850026-095f5e34-b6b8-4fab-bb9d-b6f7355086ac.png)

    

### Please specify which Issue this PR Resolves.
closes #680 
closes #1047 
closes #1686 
closes #1823 
closes #1824 
closes #2221 
closes #2222 
closes #2223 
closes #2224 
closes #2258 
closes #2287 
closes #2451 


### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Needs playtests.
********************************************************
Notes:
The Khe Sanh  cover picture missino.

Bob-Murphy commented on A3A/addons/core/functions/Base/fn_setPlaneLoadout.sqf:
```diff
@@ -256,6 +256,11 @@ if (_type == "AA") then
         {
             _loadout = ["rhs_mag_R73M_APU73","rhs_mag_R73M_APU73","rhs_mag_R73M_APU73","rhs_mag_R73M_APU73"];
         };
+        case "UK3CB_LDF_B_MIG21_AA";
```
@Lazejun this one seems to be somewhat in here because of back and for merging.
Please check

Lazejun commented on A3A/addons/maps/Antistasi_Takistan.takistan/mapInfo.hpp:
```diff
@@ -1,14 +1,16 @@
+	disabledTowns[] = {}; //no towns that need to be disabled
 	antennas[] = {
-		{4014.64,3089.66,0.150604},{5249.37,3709.48,-0.353882},{3126.7,8223.88,-0.649429},{8547.92,3897.03,-0.56073},{5578.24,9072.21,-0.842239},{2239.98,12630.7,-0.575844}
+		[[5137.9,6030.18,-0.00238037],[5578.24,9072.21,-0.842239],[5249.37,3709.48,-0.353882],[8547.92,3897.03,-0.56076],[3126.7,8223.88,-0.649429],[4014.64,3089.66,0.150574],[11710.2,9825.85,0.0984039],[2239.98,12630.7,-0.575844]]
```
Wrong Brackets here

Bob-Murphy replied:
fixed, tested and committed

Lazejun reviewed (changes requested):
(Will update the Comment as you cant directly comment on the Mission.sqms)
Altis: Most is fine, the Claimed Carrier Position change didnt happen, same pos as in 2.5.4 :heavy_check_mark:
Malden: Looks fine, Morray is disabled as claimed. :heavy_check_mark:
Tanoa: New Outpost looks fine, Stuff spawns pretty good.:heavy_check_mark:
Livonia: Carriers moved as Claimed, less than 5KM to the nearest Airbase sounds fine.:heavy_check_mark:
Stratis: Mostly for testing purposes so its fine.:heavy_check_mark:
Anizay: Slot Order got fucked, as Host/Admin Headless Clients will be the first Slots to show up instead of Green Rebells. :x:
New Antennas Positions seems fine, Fuel Stations are Spread pretty evenly 
Kunduz: Civ Pop Lowered could help with Performance.
There is 2 Antennas, one at the Northern Heli-Airport one at almost the center of the Map, closest to all Towns, for Pop support the Center one has a bit to much influence as the counter antenna is not near many towns (4 towns, there are 12 in total) :x:
Virolahti: Seems fine ✔️
Sahrani: Carrier Pos seems fine now, i havent checked all Markers for Save Spawns but "seaport_5" has a very tight Boat Spawn, That will fail on RHS, VN and anything bigger than a Speedboat 
![107410_20220917135706_1](https://user-images.githubusercontent.com/57111907/190855448-50733e14-607b-4f3e-a7dd-14e76336046c.png)
Also a lot of Outposts have Vehicle Markers on uneven Surfaces so that could cause Problems depending on Size.
❌
All Chernos: As they are roughly all the same it would apply to all of them, The CSAT Carrier Pos in the North West is way to far away from anything, to Airport_1 5539.14 M, maybe the North East Corner near the Coast is more suitable.
Also Placeholder Picture on Cherno_Autumn
Default HQ is still to Close to a Town and you will get Attacked writing a Review,
Near Zelenogorks [1752.95,5091.57,0.001297] at the Barn is pretty save, out of Spawn distance but it lacks Resources in the Area to get started.
![107410_20220917140740_1](https://user-images.githubusercontent.com/57111907/190855863-a34f698c-9f75-4a68-b492-e3ce3bac9e98.png)

Fuel seems fine, Antennas seems fine, No Banks but unsure if we ever agreed on a Suitable CUP Building yet ❌
Takistan:Pop Seems fine, rest seems fine ✔️
(Will edit this for CLN and KS, when i check them)

Bob-Murphy commented:
Kunduz antenna update
![image](https://user-images.githubusercontent.com/50651578/190875567-30e7f8a7-151e-485b-9d42-1b96b0304eca.png)

Bob-Murphy commented:
For Sahrani I basically touched every vehicle marker. Most where broken aka too small or within objects. Teleported around and the only things exploding where civ vehicles trying to spawn inside of bridges.

Bob-Murphy commented:
That should be everything addressed right now.
Please open one comment per map when you find further issues so I can answer to them.

Bob-Murphy commented:
adapted vehicle markers and antennas on malden

Bob-Murphy commented:
Also, ignore the bank mission in this overhaul.

Bob-Murphy commented:
Cherno:
Added antenna data for new tower at outpost_9
Moved HQ
Added new resource at Zelenogorsk
![image](https://user-images.githubusercontent.com/50651578/190932401-caa829a2-f826-418d-97f3-7af0629c7aca.png)


Tower coverage old
![ArmA 3 Screenshot 2022 09 17 - 17 11 51 53](https://user-images.githubusercontent.com/50651578/190932460-c5db2449-681c-4578-8566-abf743937b5c.png)


Tower coverage new
![ArmA 3 Screenshot 2022 09 19 - 01 19 26 44](https://user-images.githubusercontent.com/50651578/190932478-4d741b67-e6c1-494d-aa13-a2d16ba101c6.png)

Bob-Murphy commented:
Livonia
added a fuckton of vehicle markers and fixed broken ones
added 2 new radio towers
![image](https://user-images.githubusercontent.com/50651578/191014240-6bcd5128-59fc-4cac-b3c7-e35f0efdf9e3.png)

Bob-Murphy commented:
Regarding the init change for Khe Sanh, the whiteboard pic for Khe Sanh has to be a png or paa as there are transparent parts within the texture. That's a technical restriction based on the asset.
And CLN needs to whiteboard pic either way as that's just using the base asset without a texture applied.

Bob-Murphy commented:
Tembelan update
Added new radio tower on the small island between the others
restructured all the folders within the editor
![image](https://user-images.githubusercontent.com/50651578/192102007-6d2905e8-9c5b-47f0-ae2e-96abfd0f7589.png)

Lazejun reviewed (approved):
Havent noticed anything bad, looks fine now.

Khe Sahn still has a Preview Picture https://github.com/official-antistasi-community/A3-Antistasi/blob/b9c9891561ae67c7babbb1737199050179fd2410/A3A/addons/maps/Antistasi_vn_khe_sanh.vn_khe_sanh/Pictures/Mission/PIC.jpg

Besides that it should be good.

Lazejun commented:
The only thing thats still a Mystery is the Claimed "Moved Carrier Pos" on Altis, as said before they are still in the same spots as 2.5.5

If its all fine then this should be ready t o merge.

Bob-Murphy commented:
Tasks to be done before merge:
- check Altis for claimed carrier position changes
- add further positions in North of Khe Sanh - it's too empty

Bob-Murphy commented:
Added to Khe Sanh:
- Outpost_21
- Outpost_22
- Outpost_23
- Resource_8
- Resource_9
- a few additional vehicle markers (resources & outposts)
- a few additional mortar markers (outposts)

Bob-Murphy commented:
@Lazejun please check the last changes.
Thank you.

Lazejun reviewed (approved):
Should be good.

Also Merge conflict
